universal-login-react/src/ui/Onboarding/Onboarding.tsx

```diff
@@ -44,20 +53,15 @@ export const Onboarding = (props: OnboardingProps) => {
       contractAddress,
       privateKey
     }));
-    const relayerConfig = props.sdk.getRelayerConfig();
     const topUpProps: TopUpProps = {
       contractAddress,
       onGasParametersChanged: (parameters: GasParameters) => { gasParameters = parameters; },
       sdk: props.sdk
     };
     modalService.showModal('topUpAccount', topUpProps);
     await waitForBalance();
-    modalService.showModal('waitingFor', {
-      relayerConfig,
-      action: 'Wallet creation',
-      transactionHash: '0xee9270ccdeb9fcb92b3ec509ba11ba2362ab32ba8f...'}
-    );
-    const wallet = await walletService.deployFutureWallet(ensName, gasParameters.gasPrice.toString(), gasParameters.gasToken);
+    showWalletCreationModal('Not yet available...');
```

Comment:
The modal has an etherscan link which in this case will incorrectly navigate the user to "https://ethrscan.com/tx/Not yet available...".
Instead we should make a transaction hash optional in modal props, and make the modal use the text "Not yet available..." if `txHash === undefined`

Reply:
Good point

Reply:
👍

Reply:
Done now, looks like it was already implemented:
https://github.com/UniversalLogin/UniversalLoginSDK/blob/2adbe90b0664b484aecd79c309d6c1d6bd9fa971/universal-login-react/src/ui/commons/ExplorerLink.tsx#L11-L13